### PR TITLE
add `60_iso_tsangan_split_bs_rshift` Community Layout

### PR DESCRIPTION
**Pull request by dunk2k**
## Description

As titled

keyboards where these community layouts apply will be added in a later PR, post merge.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* n/a

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by dunk2k** [comment]
@cdarwin 
layout `60_ansi_tsangan_split_bs_rhift` is identical to `60_hhkb_tsangan` - is the `60_hhkb_tsangan` layout correct with a split Backspace and split Right Shift?

**Comment by dunk2k** [comment]
Reviewing the current `60_hhkb` community_layout reveals it's not actually true to the hhkb layout; hhkb has 6u space bar, not 7u.

Proposed name changes:
| current | new |
| :---: | :---: |
| 60_tsangan_hhkb | 60_ansi_tsangan_split_bs_rshift |
| 60_hhkb | 60_ansi_tsangan_hhkb |

File: layouts/default/60_iso_tsangan_split_bs_rshift/info.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
+{
+    "keyboard_name": "60% ISO with split Backspace, split Right Shift, and Tsangan layout",
```
Comment (noroadsleft) [inline layouts/default/60_iso_tsangan_split_bs_rshift/info.json:2]:
```suggestion
    "keyboard_name": "60% ISO layout with split Backspace, split Right Shift, and Tsangan Bottom Row",
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
I'd like to see this pared down to only the `60_iso_tsangan_split_bs_rshift` addition. The ANSI variant is a duplicate of `60_tsangan_hhkb`, and I'd rather see that renamed than to have two of the same layout with different names.

**Comment by dunk2k** [comment]
@noroadsleft 
On that note, I've changed this PR for adding `60_iso_tsangan_split_bs_rshift` only.  
Post-merge of my 60 and 64 community layouts PR's, I'll create a PR for renaming `60_tsangan_hhkb` to `60_ansi_tsangan_split_bs_rshift`.

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**